src/script/build-with-container.py

```diff
@@ -243,6 +243,12 @@ def _container_cmd(ctx, args, *, workdir=None, interactive=False):
         cmd.append(f"-eCCACHE_BASEDIR={ctx.cli.homedir}")
     for extra_arg in ctx.cli.extra or []:
         cmd.append(extra_arg)
+    if ctx.npm_cache_dir and "docker" in ctx.container_engine:
+        log.warning(
+            "The --volume option is not supported by docker build/buildx. Skipping npm cache mounts"
+        )
```

Comment:
This check is not needed, precisely because the `--volume` option is not supported for build/buildx. This function sets up arguments for a (podman|docker) run so I don't think we should block off using docker with this new option.

Reply:
I think what you're trying to say is that --volume *is* supported for docker run, and the fact that it's not supported for build/buildx is immaterial.  (I copied the dnf cache code before I realized that it affects docker build, unlike npm cache).  Will remove.